lib/tasks/evm_settings.rake

```diff
@@ -2,6 +2,7 @@ module EvmSettings
   ALLOWED_KEYS = [
     "/authentication/sso_enabled",
     "/authentication/saml_enabled",
+    "/authentication/oidc_enabled",
```

Comment:
hi Joe, with this added here we're exposing what gets shown/editable with appliance console. Do we want this enabled now or wait until the UI changes are in ?

Reply:
@abellotti Good point. I opened WIP PR https://github.com/ManageIQ/manageiq/pull/16567 to  track this change, if/when needed.